internal/pkg/cli/job_init.go

```diff
@@ -121,6 +175,110 @@ func (o *initJobOpts) Execute() error {
 	return nil
 }
 
+func (o *initJobOpts) askJobType() error {
+	o.JobType = manifest.ScheduledJobType
```

Comment:
since we don't ask for anything here, maybe instead of assigning `manifest.ScheduledJobType` to `o.JobType` here, could we do the assignment [here](https://github.com/aws/copilot-cli/pull/1390/files#diff-d5f7ba23a93d85624efe93707ecfd5e0L136) when we create the struct?

Reply:
I left this in as scaffolding for when we eventually add other job types. I've reworked the logic to add a `job-type` flag, but this method still short-circuits if the customer hasn't specified a value for the flag. See if you agree with the new changes!

Reply:
sounds good!